### PR TITLE
guide user to dmctl

### PR DESCRIPTION
**Pull request by lance6716**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-dm) that's required for repo owners to accept my contribution.

### What is changed, added, or deleted? (Required)

after deploy DM cluster, guide user to obtain `dmctl`

### Which DM version(s) do your changes apply to? (Required)

- [x] master (the latest development version, including v3.0 changes)
- [x] v2.0 (TiDB DM 2.0 versions)
- [ ] v1.0 (TiDB DM 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by CharLotteiu** [comment]
@lance6716 Please fix the CI. Thanks!

**Comment by lance6716** [comment]
> @lance6716 Please fix the CI. Thanks!

strange that I remember URL needs to start with `/` 🤔

File: zh/deploy-a-dm-cluster-using-tiup.md [hunk 1]
```diff
@@ -201,3 +201,9 @@ tiup dm display dm-test
 ```
 
 在输出结果中，如果 Status 状态信息为 `Up`，说明集群状态正常。
+
+## 第 8 步：获取集群控制工具 dmctl
+
+dmctl 是用来控制集群运行命令的工具，推荐[通过 TiUP 获取该工具](maintain-dm-using-tiup.md#%E9%9B%86%E7%BE%A4%E6%8E%A7%E5%88%B6%E5%B7%A5%E5%85%B7-dmctl)。
```
Comment (GMHDBJD) [inline zh/deploy-a-dm-cluster-using-tiup.md:207]:
```suggestion
dmctl 是用来控制集群运行命令的工具，推荐[通过 TiUP 获取该工具](maintain-dm-using-tiup.md#集群控制工具-dmctl)。
```

File: zh/deploy-a-dm-cluster-using-tiup.md [hunk 2]
```diff
@@ -201,3 +201,9 @@ tiup dm display dm-test
...
+
+dmctl 支持命令模式与交互模式，具体请见[使用 dmctl 运维集群](dmctl-introduction.md#%E4%BD%BF%E7%94%A8-dmctl-%E8%BF%90%E7%BB%B4%E9%9B%86%E7%BE%A4)。
```
Comment (GMHDBJD) [inline zh/deploy-a-dm-cluster-using-tiup.md:209]:
```suggestion
dmctl 支持命令模式与交互模式，具体请见[使用 dmctl 运维集群](dmctl-introduction.md#使用-dmctl-运维集群)。
```

**Review by GMHDBJD — COMMENTED** [review]
reset LGTM

**Review by CharLotteiu — APPROVED** [review]
LGTM

**Review by GMHDBJD — APPROVED** [review]
/lgtm

**Pull request merged**